include/Control.h

```diff
@@ -0,0 +1,8 @@
+
+#pragma once
+
+#include <glm/glm.hpp>
+
+glm::mat4 calculateMVP();
```

Comment:
I think Controls should be managed by a class (e.g., ControllerManager or InputManager) and that class should modify a Camera object (and later Geometry). A Scene would then have one or multiple cameras.